http/RemoteResource.h

```diff
@@ -26,6 +26,7 @@
 
 #ifndef  _bes_http_REMOTE_HTTP_RESOURCE_H_
 #define  _bes_http_REMOTE_HTTP_RESOURCE_H_ 1
+#define DEFAULT_EXPIRED_INTERVAL 3600
```

Comment:
This line can move to `RemoteResource.cc` which prevents it from leaking into the consumers of this class.